engines/sci/engine/kmath.cpp

```diff
@@ -206,7 +208,7 @@ reg_t kGetDistance(EngineState *s, int argc, reg_t *argv) {
 	int xdiff = (argc > 3) ? argv[3].toSint16() : 0;
 	int ydiff = (argc > 2) ? argv[2].toSint16() : 0;
 	int angle = (argc > 5) ? argv[5].toSint16() : 0;
-	int xrel = (int)(((float) argv[1].toSint16() - xdiff) / cos(angle * M_PI / 180.0)); // This works because cos(0)==1
+	int xrel = (int)(((float) argv[1].toSint16() - xdiff) / cos(Common::deg2rad<int,float>(angle))); // This works because cos(0)==1
```

Comment:
I'd like for a developer familiar with the SCI engine to indeed confirm that the double precision is unnecessary here and in the other changes below.